<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XYEquationCurveDockGeneralTab</class>
 <widget class="QWidget" name="XYEquationCurveDockGeneralTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>326</width>
    <height>518</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>2</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lName">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>23</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lComment">
     <property name="text">
      <string>Comment</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="8">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="lType">
     <property name="text">
      <string>Equation type</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lParamFuncX">
     <property name="text">
      <string>x=f(t)</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3" colspan="3">
    <widget class="KLineEdit" name="leParamFuncX"/>
   </item>
   <item row="5" column="6">
    <widget class="QToolButton" name="tbFunctions">
     <property name="toolTip">
      <string>Functions</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="7">
    <widget class="QToolButton" name="tbConstants">
     <property name="toolTip">
      <string>Constants</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lParamFuncY">
     <property name="text">
      <string>y=g(t)</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3" colspan="3">
    <widget class="KLineEdit" name="leParamFuncY">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="6">
    <widget class="QToolButton" name="tbFunctions2">
     <property name="toolTip">
      <string>Functions</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="7">
    <widget class="QToolButton" name="tbConstants2">
     <property name="toolTip">
      <string>Constants</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lParameterMin">
     <property name="text">
      <string>t, min</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="KLineEdit" name="leParameterMin">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="8" column="4" colspan="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>152</width>
       <height>18</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="lParameterMax">
     <property name="text">
      <string>t, max</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="KLineEdit" name="leParameterMax">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QLabel" name="lNumber">
     <property name="text">
      <string>Number of points</string>
     </property>
    </widget>
   </item>
   <item row="11" column="3">
    <widget class="QSpinBox" name="sbAxis1Number">
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="8">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="13" column="3" colspan="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>112</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="13" column="5" colspan="3">
    <widget class="QPushButton" name="pbRecalculate">
     <property name="text">
      <string>Recalculate</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <spacer name="verticalSpacerGeneral">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>24</width>
       <height>95</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="15" column="0" colspan="2">
    <widget class="QCheckBox" name="chkVisible">
     <property name="text">
      <string>visible</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" colspan="5">
    <widget class="KLineEdit" name="leName"/>
   </item>
   <item row="1" column="3" colspan="5">
    <widget class="KLineEdit" name="leComment"/>
   </item>
   <item row="4" column="3" colspan="5">
    <widget class="KComboBox" name="cbType">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
